examples/structured_data/movielens_recommendations_transformers.py

```diff
@@ -0,0 +1,606 @@
+"""
+Title: Recommendations with Transformer
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/12/30
+Last modified: 2020/12/30
+Description: Using the Behavior Sequence Transformer model for on Movielens dataset.
```

Comment:
Suggestion: "Build and train [model]... on [dataset]... to predict [x]"

```suggestion
Description: Build and  the Behavior Sequence Transformer (BST) model on the Movielens dataset to predict the probability of users clicking the given set of candidate items.
```